packages/cli/README.md

```diff
@@ -1,7 +1,32 @@
-## Langbase CLI
+# Langbase CLI
 
 Langbase CLI is a command line interface for Langbase. It allows you to interact with Langbase from the command line.
 
-## Documentation
+# Documentation
 
 Please follow the [Langbase documentation](https://langbase.com/docs) for the latest information.
+
+## Installation
+
+```bash
+npx @langbase/cli
```

Comment:
I don't think we need installation here. We install a package/cli via npm. npx is used to directly execute it.